HLTrigger/HLTfilters/src/HLTLevel1GTSeed.cc

```diff
@@ -348,9 +348,8 @@ bool HLTLevel1GTSeed::hltFilter(edm::Event& iEvent, const edm::EventSetup& evSet
 
         edm::ESHandle<L1GtTriggerMenu> l1GtMenu;
         evSetup.get<L1GtTriggerMenuRcd>().get(l1GtMenu);
-        // make a copy of the L1GtTriggerMenu in order to call buildGtConditionMap() (FIXME - is this really needed ?)
-        delete m_l1GtMenu;
-        m_l1GtMenu = new L1GtTriggerMenu(* l1GtMenu.product());
+        // make a copy of the L1GtTriggerMenu in order to call buildGtConditionMap()
+        m_l1GtMenu.reset(new L1GtTriggerMenu(* l1GtMenu.product()));
```

Comment:
Better to use ```std::make_unique<L1GtTriggerMenu>(*l1GtMenu.product())```. That function properly handles cleaning up memory if an exception is thrown. Not likely here, but using the function is a good habit to get into.